systemtest/src/main/java/io/strimzi/systemtest/resources/crd/kafkaclients/KafkaBasicExampleAdminClient.java

```diff
@@ -0,0 +1,258 @@
+            throw new InvalidParameterException("TopicOperation must be set.");
+        if (builder.bootstrapAddress == null || builder.bootstrapAddress.isEmpty())
+            throw new InvalidParameterException("Bootstrap server is not set.");
+        if ((builder.topicName == null || builder.topicName.isEmpty()) && !(builder.topicOperation.equals("help") || builder.topicOperation.equals("list")))
+            throw new InvalidParameterException("Topic name (or 'prefix' if topic count > 1) is not set.");
+
+        replicationFactor = (builder.replicationFactor == 0) ? 1 : builder.replicationFactor;
+        partitions = (builder.partitions == 0) ? 1 : builder.partitions;
+        topicCount = (builder.topicCount == 0) ? 1 : builder.topicCount;
+        topicOffset = builder.topicOffset;
+        adminName = builder.adminName;
+        bootstrapAddress = builder.bootstrapAddress;
+        topicName = builder.topicName;
+        additionalConfig = builder.additionalConfig;
+        namespaceName = builder.namespaceName;
+        topicOperation = builder.topicOperation;
+    }
+
+    public String getAdminName() {
+        return adminName;
+    }
+
+    public String getBootstrapAddress() {
+        return bootstrapAddress;
+    }
+
+    public String getTopicName() {
+        return topicName;
+    }
+
+    public int getTopicCount() {
+        return topicCount;
+    }
+
+    public int getTopicOffset() {
+        return topicOffset;
+    }
+
+    public String getAdditionalConfig() {
+        return additionalConfig;
+    }
+
+    public String getNamespaceName() {
+        return namespaceName;
+    }
+
+    public int getPartitions() {
+        return partitions;
+    }
+
+    public int getReplicationFactor() {
+        return replicationFactor;
+    }
+
+    public String getTopicOperation() {
+        return topicOperation;
+    }
+
+    protected KafkaBasicExampleAdminClient.Builder newBuilder() {
+        return new KafkaBasicExampleAdminClient.Builder();
+    }
+
+    protected KafkaBasicExampleAdminClient.Builder updateBuilder(KafkaBasicExampleAdminClient.Builder builder) {
+        return builder
+                .withAdditionalConfig(getAdditionalConfig())
+                .withBootstrapAddress(getBootstrapAddress())
+                .withTopicOperation(getTopicOperation())
+                .withTopicCount(getTopicCount())
+                .withTopicOffset(getTopicOffset())
+                .withPartitions(getPartitions())
+                .withReplicationFactor(getReplicationFactor())
+                .withAdminName(getAdminName())
+                .withTopicName(getTopicName())
+                .withNamespaceName(getNamespaceName());
+    }
+
+    public KafkaBasicExampleAdminClient.Builder toBuilder() {
+        return updateBuilder(newBuilder());
+    }
+
+    public JobBuilder adminStrimzi() {
+        return defaultAdminStrimzi();
+    }
+
+    public JobBuilder defaultAdminStrimzi() {
```

Comment:
Curious why is `defaultAdminStrimzi` the right name here. From looking into the method, I can see that it gives you _Admin Client_ with some default settings ... But what does the Strimzi in the name mean?

Reply:
Same like in comment below -
```
public JobBuilder producerStrimzi() {
        return defaultProducerStrimzi();
    }

    public JobBuilder consumerStrimzi() {
        return defaultConsumerStrimzi();
    }
```

Reply:
Well, and why is it in the names below? :-o

Reply:
We will rename it in other classes in different PR, I don't remember why we named it like this in past.  Can you change it in the AdminClient class @michalxo ?

Reply:
Done